Starter_Code/assign-1-life-updated/assign-1-life-updated/src/life.cpp

```diff
@@ -31,14 +33,68 @@ static void welcome() {
     getLine("Hit [enter] to continue....   ");
 }
 
+static string promptConfigFile() {
+    string fileName = getLine("Please enter a config file [press enter if no Config]:");
+    while (!fileExists(fileName) || fileName != "") { //unsure why this does not work gets trapped in loop
```

Comment:
use a half while loop structure, it tends to be more clear:

```
while (true) {
    string fileName = getLine("Error: File does not exist. Please enter a config file [press enter if no Config]: ");
    if (fileExists(fileName) || fileName == "") {
        return fileName
    }
}
```

Reply:
will do